shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -779,6 +784,93 @@ void onBackPressed() {
     }
   }
 
+  /**
+   * Invoke this from {@link OnBackAnimationCallback#onBackStarted(BackEvent)}.
+   *
+   * <p>This method should be called when the back gesture is initiated, as part of the
+   * implementation of {@link OnBackAnimationCallback}. It's responsible for handling the initial
```

Comment:
Nit: can this doc is some way clarify that what it's responsible for is defined in the framework?

Reply:
Same nit for the other methods you added here